test/index.js

```diff
@@ -1,245 +0,0 @@
-require('should');
-var Limiter = require('..');
```

Comment:
I'd change the name of new get to something else, like `count` and deprecate the `get` as well, but still support the old API. This is the best approach as I want to believe that this library is considered stable.
does it sound good?